### ISSUE TITLE
Bounds for Ghost (not too far out of scene)

### ISSUE THREAD

**Issue by bradleyknaysi** [petertran-21/untitled-ghost-game#36]
# Potential Solution
Add metadata to the scene json that sets global boundaries

**Comment by jliao13** [comment]
Design team has been considering the problem that the ghost can travel as far out into space as it wants. I'm not sure if that applies to this.

**Comment by bradleyknaysi** [comment]
Yeah that's the issue i'm referring. I think a good solution would be to save scene dimensions in the `.json` files... but I can work on this
